std_msgs/Header header
stargazer/MarkerRawPose[] marker_poses